tox.ini

```diff
@@ -1,11 +1,15 @@
 [tox]
-envlist = py39
+envlist = py
 tox_pip_extensions_ext_venv_update = true
 tox_pip_extensions_ext_pip_custom_platform = true
 
 [testenv]
+description = Run unittests and linters
 deps = -rrequirements-dev.txt
-passenv = HOME SSH_AUTH_SOCK USER
+passenv =
+    HOME
+    SSH_AUTH_SOCK
+    USER
```

Comment:
this can just be removed

Reply:
Kinda true but if we were to correct, it would be safe to remove only if we also require a newer version of tox, as these vars were whitelisted at some point in time. Removing the entire set w/o adding a min version, would risk on introducing a regression for someone using a very old version of tox.

Is ok to add a minversion = 4.0 ? WDYT?